src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Ball {
+    private ColorSupplier.Color color;
+    private int number;
+
+    public Ball(ColorSupplier.Color color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+    //getter and setter to color
+    public ColorSupplier.Color getColor() {
+        return color;
+    }
+
+    public void setColor() {
+        this.color = color;
+    }
+    //getter and setter to number
+    public int getNumber() {
+        return number;
+    }
+
+    public void setNumber() {
+        this.number = number;
```

Comment:
The 'setNumber' method does not take any parameters. It should take an 'int' parameter to set the number of the ball.